modules/phase_field/src/postprocessors/FeatureFloodCount.C

```diff
@@ -75,9 +65,9 @@ void dataLoad(std::istream & stream, FeatureFloodCount::FeatureData & feature, v
 }
 
 template<>
-void dataLoad(std::istream & stream, MooseSharedPointer<FeatureFloodCount::FeatureData> & feature, void * context)
+void dataLoad(std::istream & stream, std::unique_ptr<FeatureFloodCount::FeatureData> & feature, void * context)
```

Comment:
I believe I got compile errors without it. What are you asking about? The line here, or the one in DataIO.h?